sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -50,60 +189,52 @@ namespace Azure { namespace Core {
       /// RFC 1123.
       Rfc1123,
 
-      /// ISO 8601.
-      Iso8601,
+      /// RFC 3339.
+      Rfc3339,
     };
 
     /**
      * @brief Get the current UTC time.
      */
-    static DateTime UtcNow();
-
-    /// An invalid UTC timestamp value.
-    static constexpr IntervalType UtcTimestampInvalid = static_cast<IntervalType>(-1);
+    static DateTime Now();
 
     /**
-     * @brief Get seconds since Unix/POSIX time epoch at `01-01-1970 00:00:00`.
-     * If time is before epoch, @UtcTimestampInvalid is returned.
-     */
-    static IntervalType UtcTimestamp()
-    {
-      auto const seconds = UtcNow().ToInterval() / WindowsToPosixOffsetSeconds;
-      return (seconds >= WindowsToPosixOffsetSeconds) ? (seconds - WindowsToPosixOffsetSeconds)
-                                                      : UtcTimestampInvalid;
-    }
-
-    /**
-     * @brief Construct an uninitialized (!@IsInitialized()) instance of @DateTime.
+     * @brief Construct an instance of @DateTime.
+     *
+     * @param year Year.
+     * @param month Month.
+     * @param day Day.
+     * @param hour Hour.
+     * @param minute Minute.
+     * @param seconds Seconds.
+     *
+     * @throw DateTimeException If any paramter is invalid.
      */
-    DateTime() : m_interval(0) {}
+    DateTime(int year, int month = 1, int day = 1, int hour = 0, int minute = 0, int seconds = 0);
```

Comment:
Why are all parameters singular except for seconds?

```suggestion
    DateTime(int year, int month = 1, int day = 1, int hour = 0, int minute = 0, int second = 0);
```